doc/release-notes-6187.md

```diff
@@ -0,0 +1,5 @@
+## Devnet Breaking Changes
+
+Many hard-forks have been delayed to block-300 on devnet to be compatible with existing devnets.
+
+Let's make a breaking changes (for devnet) to activate everything as fast as they are ready, just from block 2.
```

Comment:
how about smth like
```suggestion
`BRR` (`realloc`), `DIP0020`, `DIP0024`, `V19`, `V20` and `MN_R` hardforks are activated at block 2 instead of block 300 on devnets now. Please note that this is a breaking change so you will no longer be able to sync on devnets created with earlier Dash Core versions and vice versa.
```
?